app/main.py

```diff
@@ -1,25 +1,77 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Any, name: Any) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: Any, owner: Any) -> None:
```

Comment:
The return type of the `__get__` method should not be `None`. It should be `Any` or the expected type of the attribute being accessed, as this method returns the value of the attribute.